addons/udes_transport_management/models/transport.py

```diff
@@ -59,29 +59,38 @@ def get_info(self, **kwargs):
         return res
 
     def prepare_transport_info(self, **kwargs):
```

Comment:
This function will be removed, since it does not belong to this module.

Reply:
@ws-mps All done